tokens/build.mjs

```diff
@@ -0,0 +1,84 @@
+import StyleDictionaryPackage from 'style-dictionary'
+const { fileHeader, formattedVariables } = StyleDictionaryPackage.formatHelpers
+
+const themes = ['Reapit', 'PayProp']
+
+const getStyleDictionaryConfig = (theme) => {
+  const lowerCasedTheme = theme.toLowerCase()
+  return {
+    source: ['./tokens/tokens.json'],
+    format: {
+      linariaFormat: ({ dictionary, options, file }) => {
+        const { outputReferences } = options
+        const lineSeparator = '\n'
+
+        // Gives us a template literal of CSS variables we can import into linaria
+        return (
+          fileHeader({ file }) +
+          `export const tokens${theme} = ` +
+          '`' +
+          lineSeparator +
+          formattedVariables({
+            dictionary,
+            outputReferences,
+            format: 'css',
+            formatting: {
+              prefix: '--',
+              separator: ':',
+              suffix: ';',
+            },
+          }) +
+          lineSeparator +
+          '`' +
+          lineSeparator
+        )
+      },
+    },
+    parsers: [
+      {
+        pattern: /\.json$/,
+        parse: ({ contents }) => {
+          try {
+            const object = JSON.parse(contents)
+            // Bit fragile but I only want the primitives, semantic and component variables for each theme
+            // Figma exports a "mode 1" that we don't need - confirmed with Andrei
```

Comment:
**praise:** It may be a bit fragile, but at least is well-contained to this file 👌 👏